easybuild/framework/easyconfig/tools.py

```diff
@@ -344,8 +344,9 @@ def det_easyconfig_paths(orig_paths):
 
     if from_pr_list is not None:
         pr_files = []
+        tmpdir = tempfile.mkdtemp()
         for pr in from_pr_list:
-            pr_files.extend(fetch_easyconfigs_from_pr(pr))
+            pr_files.extend(fetch_easyconfigs_from_pr(pr, path=os.path.join(tmpdir, 'files_pr%s' % pr)))
```

Comment:
Just changing this here is not enough, and even incorrect (that's why you found yourself having to add the `OpenBLAS` easyconfig to the tests to make things work).

There's a hidden shared path between `alt_easyconfig_paths` (which was tweaked in #3605 to take into account multiple PR #s, so you get a path like `/.../files_pr123_456` when using `--from-pr 123,456`), which then gets set as the value for the `pr_path` build option. This is then used in `fetch_files_from_pr` to determine the default value for the easyconfigs being pulled from a PR (which you're overriding here).

So if we spread easyconfigs from PRs over multiple paths, we have to take that into account in `alt_easyconfig_paths` too, or we ensure that all those paths get injected into the robot search path correctly (which is done via the result produced by `det_robot_path` in `set_up_configuration`)...